src/components/TransitionAnimation/story.tsx

```diff
@@ -20,7 +20,13 @@ storiesOf('TransitionAnimation', module).add('Default', () => {
         <Box direction="column" alignItems="center">
```

Comment:
removing alignItems makes for a better story imo :)

```suggestion
         <Box direction="column">
```